Content.Server/StationEvents/Events/DiseaseOutbreak.cs

```diff
@@ -32,16 +33,24 @@ public sealed class DiseaseOutbreak : StationEvent
     public override float Weight => WeightNormal;
     protected override float EndAfter => 1.0f;
     /// <summary>
-    /// Finds 2-5 random entities that can host diseases
+    /// Finds 2-5 random, alive entities that can host diseases
     /// and gives them a randomly selected disease.
     /// They all get the same disease.
     /// </summary>
     public override void Startup()
     {
         base.Startup();
 
-        var targetList = _entityManager.EntityQuery<DiseaseCarrierComponent>().ToList();
+        List<(DiseaseCarrierComponent carrier, MobStateComponent mobState)> targetList =
+            _entityManager.EntityQuery<DiseaseCarrierComponent, MobStateComponent>().ToList();
         _random.Shuffle(targetList);
+        /// We're going to filter the above out to only alive mobs. Might change after future mobstate rework
+        List<DiseaseCarrierComponent> aliveList = new();
+        foreach (var target in targetList)
+        {
+            if (!target.mobState.IsDead())
+                aliveList.Add(target.carrier);
+        }
```

Comment:
This is going to allocate 2 lists when you should just enumerate over the entityquery.